### PR TITLE
Experiment improvements

### PR DESCRIPTION
**Pull request by bhufmann**
- Create an experiment from list of trace paths.
- Delete experiments also deletes traces when a certain flag is passed.

### CODE REVIEW

File: tsp-cli-client [hunk 1]
```diff
@@ -152,19 +155,38 @@ if __name__=="__main__":
...
-                response = tsp_client.open_experiment(options.experiment, options.uuids)
-                if response.status_code == 200: 
-                    res = response.model
-                    print_experiment(res)
-                    sys.exit(0)
-                else:
-                    sys.exit(1)
+            if options.paths is not None:
+                for path in options.paths:
+                    response = tsp_client.open_trace(path, path)
```
Comment (bhufmann) [inline tsp-cli-client:170]: we should use the last segment as name otherwise the server resource path would be too long.

File: tsp-cli-client [hunk 2]
```diff
@@ -104,6 +105,8 @@ if __name__=="__main__":
     parser.add_argument("--get-tree", dest="get_tree", help="Gets the tree of an output", metavar="OUTPUT_ID")
     parser.add_argument("--uuid", dest="uuid", help="The UUID of a trace")
     parser.add_argument("--uuids", dest="uuids", help="The list of UUIDs", nargs="*")
+    parser.add_argument("--do-delete-traces", dest="do_delete_traces", action='store_true', help="Also delete traces when deleting experiment")
+    parser.add_argument("--paths", dest="paths", help="Also delete traces when deleting experiment", nargs="*")
```
Comment (PatrickTasse) [inline tsp-cli-client:109]: The help is wrong?

File: tsp-cli-client [hunk 3]
```diff
@@ -156,22 +159,34 @@ if __name__=="__main__":
...
+                    response = tsp_client.open_trace(name, path)
+                    if response.status_code == 200: 
+                        res = response.model
+                        trace_uuids.append(res.UUID)
+                    else:
+                        sys.exit(1)
+            elif options.uuids is not None:
+                trace_uuids = options.uuids;
+            else:
+                print("Provide a path for the trace using option --path")
```
Comment (PatrickTasse) [inline tsp-cli-client:181]: Should it be --paths ?

**Review by PatrickTasse — CHANGES_REQUESTED** [review]

File: tsp-cli-client (hunk 2 above)
Comment (bhufmann) [reply tsp-cli-client:109]: true. I'll fix it.

File: tsp-cli-client (hunk 3 above)
Comment (bhufmann) [reply tsp-cli-client:181]: also true. I'll fix it

**Review by PatrickTasse — APPROVED** [review]

**Pull request merged**